modules/compose/templates/docker-compose-container.yml.erb

```diff
@@ -0,0 +1,150 @@
+# ###################################
+# This file is generated by puppet
+# PLEASE DON'T MODIFY BY HAND
+# ###################################
+version: '2'
+services:
+
+
+########################
```

Comment:
I'm not sure that it will be a real debugging if you have 2 different (even slightly) compose files and they will be executed in a different ways